app/javascript/stylesheets/application.scss

```diff
@@ -0,0 +1,86 @@
+@import "bootstrap"; 
+@import '@fortawesome/fontawesome-free';
+
+body{
+background-color: #eee9e1;
+}
+
+.header{
+background-image: url("../../../public/bookshelf.jpg");
+background-repeat: no-repeat;
+background-attachment: fixed;
+background-size: 100%;
+height: 235px;
+}
+
+.navbar-brand{
+font-weight: bold;
+font-size: x-large;
+}
+
+.auth-btn {
+color: whitesmoke
+}
+
+.auth-btn:hover {
+color: whitesmoke;
+background-color: #742d05;
+}
+
+.navbar-header{
+background-color: rgba(255, 255, 255, 0.171);
+}
+
+.nav-links {
+color: white;
+}
+
+.nav-links:hover{
+color: whitesmoke;
+background-color: #742d05;
+}
+
+.footer{
```

Comment:
move to a separate file(the same about `.imd-book`)
keep here the general for the whole project things